parachains/common/src/xcm_config.rs

```diff
@@ -82,24 +82,30 @@ pub struct AssetFeeAsExistentialDepositMultiplier<Runtime, WeightToFee, BalanceC
 impl<CurrencyBalance, Runtime, WeightToFee, BalanceConverter>
 	cumulus_primitives_utility::ChargeWeightInFungibles<
 		AccountIdOf<Runtime>,
-		pallet_assets::Pallet<Runtime>,
+		pallet_assets::Pallet<Runtime, pallet_assets::Instance1>,
 	> for AssetFeeAsExistentialDepositMultiplier<Runtime, WeightToFee, BalanceConverter>
 where
-	Runtime: pallet_assets::Config,
+	Runtime: pallet_assets::Config<pallet_assets::Instance1>,
 	WeightToFee: WeightToFeePolynomial<Balance = CurrencyBalance>,
 	BalanceConverter: BalanceConversion<
 		CurrencyBalance,
-		<Runtime as pallet_assets::Config>::AssetId,
-		<Runtime as pallet_assets::Config>::Balance,
+		<Runtime as pallet_assets::Config<pallet_assets::Instance1>>::AssetId,
+		<Runtime as pallet_assets::Config<pallet_assets::Instance1>>::Balance,
```

Comment:
How can I make this generic over any instance of `pallet_assets`?

Reply:
Can you not do it similar to the `AssetsToBlockAuthor` by adding a `AssetInstance: 'static` generic?